tests/unit/lxd/test_launcher.py

```diff
@@ -212,19 +236,46 @@ def test_launch_use_base_instance(
     ]
     assert fake_instance.mock_calls == [
         call.exists(),
-        call.launch(image="image-name", image_remote="image-remote", ephemeral=False),
-        call.stop(),
         call.start(),
     ]
     fake_base_instance.exists.assert_called_once()
     assert mock_base_configuration.mock_calls == [
         call.get_command_environment(),
         call.get_command_environment(),
-        call.setup(executor=fake_instance),
+        call.setup(executor=fake_base_instance),
         call.wait_until_ready(executor=fake_instance),
     ]
 
 
+def test_launch_use_base_instance_failed_lxc(
+    fake_instance,
+    fake_base_instance,
+    mock_base_configuration,
+    mock_is_valid,
+    mock_lxc,
+    mock_lxd_instance,
+    mock_platform,
+    mock_timezone,
+):
+    """Launch a base instance from an image, but lxc commands fail."""
+    mock_lxc.config_set.side_effect = [
+        LXDError("test1"),
+        LXDError("test2"),
+        LXDError("test3"),
+    ]
```

Comment:
This test only calls `config_set` once before the LXDError is raised so why specify 2 more side_effects that are unused?

Reply:
It's a leftover from my intermediate tests